interpreter/cling/lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -420,7 +420,7 @@ namespace {
     R.clear();
     R.setLookupName(&m_Context->Idents.get("copyArray"));
     m_Sema->LookupQualifiedName(R, NSD);
-    assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
+    //assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
```

Comment:
Can you add a `//FIXME` with some text why this is commented, and when it should be re-enabled? Can we make this conditional, e.g. "only assert if we have no parent interpreter"?